com.unity.cinemachine/Samples~/samples.json

```diff
@@ -0,0 +1,31 @@
+{
+  "SharedAssetDependencies": [
+    "Shared"
+  ],
+  "SampleEntries": [
+    {
+      "Path": "2D Samples",
+      "AssetDependencies": [],
+      "PackageDependencies": []
+    },
+    {
+      "Path": "3D Samples",
+      "AssetDependencies": [],
+      "PackageDependencies": []
+    },
+    {
+      "Path": "Physics Samples",
```

Comment:
I think this is not a good idea.  Our shared scripts have physics dependencies (and there are colliders in the non-physics scenes).  I think it's easier just to accept that our samples all depend on physics.

Reply:
Yes, I agree. I mention that in the Comments to Reviewers part. I removed Physics samples in the next PR - https://github.com/Unity-Technologies/com.unity.cinemachine/pull/665.